Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:jessie
+MAINTAINER projectcalico
+
+ADD ./dist /dist
```

Comment:
Sorry, very possible I'm just being daft but I want to understand a few things here:

- Why does this place the binaries in `/usr/sbin` rather than `/opt/cni/bin`, which is the default location for CNI binaries?
- Why does this install the binaries in two steps, rather than one? e.g `ADD ./dist/calico /opt/cni/bin`
- What is the significance of the `sync` - is that necessary?